### PR TITLE
a super cool announcement

### PR DESCRIPTION
**Pull request by ghuntley**

### CODE REVIEW

File: src/routes/blog/gitpod-open-source-sustainability-fund.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+4. ⒋ Over the coming months, Gitpod will share our experiences with funding open-source projects for other companies to follow and share insights for maintainers who wish to become financially independent.
+
+We recognise that money <a href="https://www.youtube.com/watch?v=Mm_RuObpeGo">isn't the only way to resolve open-source sustainability</a>. Often the problem is access and the initial project setup friction. To address this Gitpod has made <a href="/docs/professional-open-source/">Gitpod free for open-source communities</a> so that open-source maintainers to onboard new contributors to their project with a standardized development environments. People from all over the world can contribute from any device without the typical $5,000 Macbook Pro barrier to entry.
```
Comment (ghuntley) [inline src/routes/blog/gitpod-open-source-sustainability-fund.md:56]: to -> can

File: src/routes/blog/gitpod-open-source-sustainability-fund.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+`youtube: 0t85TyH-h04`
+
+Much has changed since Russells' keynote and the infrastructure to financially support open-source maintainers now exists thanks to the efforts of Open Collective and GitHub Sponsors.
+
+# Shape the future you want to live in
+
+The truth is free software isn't free. How much is a task going to cost in person hours? Hours have rates. Rates require payment. Yet, an entire community of developers provide their services outside of their day job to produce tools consumed by businesses.
+
+Did you know there are less than 2100 people in the inner GitHub open-source maintainer community?
```
Comment (ghuntley) [inline src/routes/blog/gitpod-open-source-sustainability-fund.md:30]: Whilst the open-source ecosystem is bigger than GitHub.

File: src/routes/blog/gitpod-open-source-sustainability-fund.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+At Gitpod, we firmly believe that working for an open-source project should be a legitimate alternative to a career working for a for-profit corporation. Additionally, we'd like everyone to be able to maintain open-source projects, not just people with the right financial background or youth that enables the ability to volunteer.
+
+# Actions speak louder than words
+
+Today we are announcing four things:
+
+1. ⒈ Gitpod has created an open-source sustainability fund and allocated an _initial_ amount of USD 30,000 has been allocated towards securing our open-source supply chain by paying open-source maintainers.
+
+2. ⒉ For every registration to our newly created <a href="https://www.gitpod.io/chat">discord channel</a> we’ll donate an additional $1 to open-source projects we depend upon.
```
Comment (ghuntley) [inline src/routes/blog/gitpod-open-source-sustainability-fund.md:50]: During the next month.

File: src/routes/blog/gitpod-open-source-sustainability-fund.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+![Open-source supply chain decision tree](../../../static/images/blog/gitpod-open-source-sustainability-fund/teaser.jpg)
+
+At Gitpod, we firmly believe that working for an open-source project should be a legitimate alternative to a career working for a for-profit corporation. Additionally, we'd like everyone to be able to maintain open-source projects, not just people with the right financial background or youth that enables the ability to volunteer.
+
+# Actions speak louder than words
+
+Today we are announcing four things:
+
+1. ⒈ Gitpod has created an open-source sustainability fund and allocated an _initial_ amount of USD 30,000 has been allocated towards securing our open-source supply chain by paying open-source maintainers.
```
Comment (ghuntley) [inline src/routes/blog/gitpod-open-source-sustainability-fund.md]: double allocated

File: src/routes/blog/gitpod-open-source-sustainability-fund.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+Gitpod hopes other companies will also start taking steps to mitigate this critical risk in their technology stack. In future blog posts, we will be sharing our experiences with funding open-source projects and insights for maintainers who wish to become financially independent.
+
+## Towards a maintainer economy
+
+Here is what you can do today:
+
+<ol>
+    <li>
+      <span>1- </span>Generate the bill of materials in your commerical software.
```
Comment (mikenikles) [inline src/routes/blog/gitpod-open-source-sustainability-fund.md:102]: We have ordered lists styled consistently across the website with CSS. Is there a particular reason we want this to look differently here?

If not, this HTML `ol` can be replaced with an ordered list written in Markdown.

**Review by mikenikles — CHANGES_REQUESTED** [review]
I notice `<a>` and `<ol>` HTML tags in the blog's Markdown. Let's use Markdown syntax unless there is a good reason to inject raw HTML, e.g. if we have to add a `class` attribute to style something in a particular way.

The `OSS_fund_header.jpg` image needs to be moved to `static/images/blog/gitpod-open-source-sustainability-fund` so that the cover image displays correctly. This will also fix the broken image on the actual blog post itself and in fact, the Netlify deployment.

Please remove `yarn.lock`. I created #405 to ignore that file and to document the use of NPM for this project.

**Comment by JohannesLandgraf** [comment]
@ghuntley looks good some minor things: 

- [ ] Replace the teaser image with a new background. Updated [here](https://www.figma.com/file/Mlo0SjCChzsbSiCWMP4BzP/OSS-Fund-Blog-Post?node-id=1%3A40).
- [ ] tandardized -> standardized
- [ ] There are now 5 not four things :P

**Pull request merged**